ports/glutin/headless_window.rs

```diff
@@ -157,56 +109,39 @@ impl WindowPortsMethods for Window {
         self.animation_state.get() == AnimationState::Animating
     }
 
-    fn winit_event_to_servo_event(&self, _event: glutin::WindowEvent) {
+    fn winit_event_to_servo_event(&self, _event: winit::WindowEvent) {
```

Comment:
Would it be worthwhile to put an ```unreachable!()``` here?

Reply:
Possibly? I'm trying to keep this PR as small as I can and not do too many drive-by fixes.